spec/services/gpt_service/generate_tests_spec.rb

```diff
@@ -65,7 +65,9 @@
         test_file = task.reload.tests.last.files.first
         expect(test_file).to have_attributes(
           content: "def test_script():\n  assert true",
-          name: 'test.py'
+          name: 'test.py',
+          visible: 'yes',
```

Comment:
As above, I would opt to hide a test by default.

Reply:
Thanks. I have reverted to `no` and will push now.